AI/StupidAI/StupidAI.cpp

```diff
@@ -49,7 +49,8 @@ struct EnemyInfo
 	const CStack * s;
 	int adi, adr;
 	std::vector<BattleHex> attackFrom; //for melee fight
-	EnemyInfo(const CStack * _s) : s(_s), adi(0), adr(0)
+	EnemyInfo(const CStack * _s)
+		: s(_s), adi(0), adr(0)

```

Comment:
New line for everything sounds like a good option for me. And normally you don't need a struct with one member so you always end up with 2+ members